mlir-tensorrt/compiler/lib/Compiler/StableHloToExecutable.cpp

```diff
@@ -482,6 +436,7 @@ StableHloToExecutableTask::compileStableHLOToExecutable(
 //===----------------------------------------------------------------------===//
 
 namespace {
+// TODO (pranavm): Any way to reuse `DeviceOptions` here?
```

Comment:
Not quite sure how to deduplicate `ClusteringPipelineCliOpts` and `DeviceOptions`

Reply:
A million dollar question :joy:. In all seriousness, I don't think you can right now. We're butting up against limitations of C++ (no builtin reflection) and the design of the upstream `PassPipelineOptions` (no inheritance scheme/limited flexibility). You would need to make the upstream `PassPipelineOptions` more flexible at least.

Reply:
Ah that's unfortunate. Will leave it as-is for now.